src/utils/decancer.rs

```diff
@@ -9,6 +10,7 @@ pub async fn cure(
     old_if_available: &Option<serenity::Member>,
     member: &serenity::Member,
 ) {
+    let censor = Standard + "nigga";
```

Comment:
What does this line do?

Reply:
The Censor crate features 3 bundled wordlists

The `let censor = Standard` part is me adding their `Standard` wordlist

The addition that follows adds the n word into our list as it isn't included in the `Standard` wordlist by default 

more details here https://docs.rs/censor/0.3.0/censor/

Reply:
Anything outside of the standard lists should be configured through `configuration.json` and not hardcoded into the app.

Reply:
I will try to do that tonight after work then